src/content/docs/style-guide/writing-strategies/install-docs-guidance.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Choose from our install methods
+topics:
+  - New Relic only
+  - Basic style guide
+  - Writing guidelines
+---
+
+Installing New Relic is a crucial piece in our customer's experience. A failure to install something not only prevents our customers from using New Relic, but it almost certainly guarantees they won't be our customers eventually. Whether it's installing APM agents, the infrastructure agent, sending us logs, or installing CodeStream, among other things.
```

Comment:
The second sentence feels incomplete to me.

Reply:
I deleted it, because I decided I didn't like it :D